packages/services/server/src/graphql-handler.ts

```diff
@@ -165,15 +179,21 @@ export const graphqlHandler = (options: GraphQLHandlerOptions): RouteHandlerMeth
       }),
       useSentryUser(),
       useErrorHandler(({ errors, context }): void => {
+        // Not sure what changed, but the `context` is now an object with a contextValue property.
+        // We previously relied on the `context` being the `contextValue` itself.
+        const ctx = ('contextValue' in context ? context.contextValue : context) as Context;
+
```

Comment:
@ardatan any idea why? I thought `context` === `contextValue` but apparently it's not.